### PR TITLE
Update OSS documentation and guidelines for contribution

### PR DESCRIPTION
**Pull request by ppbaena**
### Description of the change

Edit project documentation to improve the community experience through Kubeapps documentation

### Benefits

Outline main guidelines for contributors and help users to explore Kubeapps documentation

### Possible drawbacks

It will be necessary to review, and update if necessary, some technical documentation to be aligned with the OSS guidelines and best practices

### Applicable issues

#2388 

### Additional information

It is a starting point to improve Kubeapps documentation, so more updates will be necessary but it is necessary to engage our community and increase the relationship with other contributors.

### CODE REVIEW

**Comment by ppbaena** [comment]
Related to #2388

File: README.md
Comment (absoludity) [inline README.md:30]: Just a small tweak - feel free to ignore.
```suggestion
> Kubeapps is deployed using the official [Bitnami Kubeapps chart](https://github.com/bitnami/charts/tree/master/bitnami/kubeapps) from the separate Bitnami charts repository. Although the Kubeapps repository also defines a chart, this is intended for development purposes only.
```

File: README.md
Comment (absoludity) [inline README.md:56]: Woo - breaking away from the Monocular past :P

**Review by absoludity — APPROVED** [review]
> Share this draft version to receive some feedback and discuss about it

I like it :) Much simpler and concise. About time we removed the references to monocular (which is no longer used since helm hub is now pointing at artifacthub).

File: README.md
Comment (antgamdia) [inline README.md:13]: Perhaps it worths adding some links here?
```suggestion
- Browse and deploy from public or private chart repositories (including [VMware Marketplace™](https://marketplace.cloud.vmware.com) and [Bitnami Application Catalog](https://bitnami.com/application-catalog))
```

File: README.md
Comment (antgamdia) [inline README.md:15]: Perhaps I'd also mention here something about Pinniped like:

```suggestion
- Secure authentication to Kubeapps using a [standalone OAuth2/OIDC provider](./docs/user/using-an-OIDC-provider.md) or [using Pinniped](./docs/user/using-an-OIDC-provider-with-pinniped.md)
```

File: README.md
Comment (antgamdia) [inline README.md:41]: I'd rather remove them from this section and will wait until we have a chance to revisit them. Later, I'd add these links in a `## Contributing` section (along with the contributing.md, and other files)

File: README.md
Comment (antgamdia) [inline README.md:59]: Not sure about the wording here, "_to find out about feature changes_" -> I'd change it to include also bug fixes (I mean, we also publish patch versions without new features).
So sth like

```suggestion
Take a look at the list of [releases](https://github.com/kubeapps/kubeapps/releases) to stay tuned for the latest features and changes.
```

But don't know; as it is the last sentence, we need a sort of a powerful finale

**Review by antgamdia — COMMENTED** [review]
Great! Big improvements here! Thank you for revamping the readme!!

File: MAINTAINERS.md
Comment (absoludity) [inline MAINTAINERS.md:7]: FWIW, happy for alphabetical ordering here :)

File: ADOPTERS.md
Comment (absoludity) [inline ADOPTERS.md:7]: If you check the result at https://github.com/ppbaena/kubeapps/blob/oss-documentation-update/ADOPTERS.md , you'll see that as an SVG with no dimensions, it's just going to scale to width, which we may not want. AFAICT, that means using an img element instead. See https://gist.github.com/absoludity/739192bb1d6da37faf1329f81d1fb568 for an example.


Also, I think the text `logo` should clearly identify the organization instead?

File: ADOPTERS.md
Comment (absoludity) [inline ADOPTERS.md:15]: What is XXX here? Could be "unofficial", not sure what was intended.

File: CODE_OF_CONDUCT.md
Comment (absoludity) [inline CODE_OF_CONDUCT.md:40]: Just be aware here and throughout, it looks like the original text was using 80char lines. Don't care either way, but want to be consistent in the one doc. I'd just follow the decision of the original markdown which appears to be 80 char lines (see https://www.contributor-covenant.org/version/2/0/code_of_conduct/code_of_conduct.md )

File: CODE_OF_CONDUCT.md
Comment (absoludity) [inline CODE_OF_CONDUCT.md:88]: Broken link. Looks like they removed the .html for the latest version.
```suggestion
available at https://www.contributor-covenant.org/version/2/0/code_of_conduct/
```

File: GOVERNANCE.md
Comment (absoludity) [inline GOVERNANCE.md:1]: I was looking for a link to something on which this governance doc is based? Is there one? It's quite interesting, and worth indicating the source if so, imo.

File: README.md
Comment (antgamdia) [inline README.md:14]: The second sentence already includes the first one, no? 
What about this?

- Browse and deploy [Helm](https://github.com/helm/helm) charts from public or private chart repositories (including [VMware Marketplace™](https://marketplace.cloud.vmware.com) and [Bitnami Application Catalog](https://bitnami.com/application-catalog))

File: CONTRIBUTING.md
Comment (antgamdia) [inline CONTRIBUTING.md:3]: I'd add here that we have delegated the chart distribution to the official BAC and thereby the PRs and issues should be created in the bitnami/charts repo.

File: CODE_OF_CONDUCT.md
Comment (absoludity) [reply CODE_OF_CONDUCT.md:40]: Yeah, I don't care either way, which is why I'd defer to the decision upstream had made (not sure if their reasons, but I'd guess it wasn't related to punch-cards). That said, another way to look at consistency is that all *our* docs (regardless of where they came from) use the same. +1 for natural lines in that case.

Comment (ppbaena) [reply CODE_OF_CONDUCT.md:40]: Agree. Keep with natural lines

File: CONTRIBUTING.md
Comment (ppbaena) [reply CONTRIBUTING.md]: Thank you. I've reviewed absolute links and turned into relative ones.

File: GOVERNANCE.md
Comment (ppbaena) [reply GOVERNANCE.md:30]: Indeed. Main goal is to be open to the community so information about new maintainers will be shared and homogeneous for any contributor (even if it is someone hired by our company).

File: GOVERNANCE.md
Comment (ppbaena) [reply GOVERNANCE.md:44]: We can remove the sentence about our roadmap, because it does not apply. It is interesting to upload our own proposals of evolution to the repo, as well as any proposal from the community.

File: GOVERNANCE.md
Comment (ppbaena) [reply GOVERNANCE.md:1]: Added some credits. I saw other oss projects in Github doing so.

**Comment by absoludity** [comment]
I like the governance details too. Still +1 from me. @antgamdia , are there still changes you're keen to see here or just did you just forget to +1?

**Review by antgamdia — APPROVED** [review]
Oops, I forgot to +1 it. Sorry! 
Thanks again for all the changes. Happy to see how we improve and adopt OSS best practices :)

**Comment by antgamdia** [comment]
Crossposting for transparency what Michael commented:
https://tanzu.vmware.com/content/blog/how-to-build-strong-open-source-community-best-practices-lessons-learned

**Pull request merged**
